src/test/java/uk/gov/hmcts/reform/unspec/handler/CreateClaimCallbackHandlerTest.java

```diff
@@ -32,79 +33,98 @@
-
-        AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
-
-        assertThat(response.getErrors()).isEmpty();
-        assertThat(response.getData())
-            .isEqualTo(
-                Map.of(
-                "claimValue", ClaimValue.builder()
-                        .higherValue(BigDecimal.valueOf(10))
-                        .lowerValue(BigDecimal.valueOf(1))
-                        .build(),
-                "claimType", PERSONAL_INJURY_WORK,
-                "allocatedTrack", SMALL_CLAIM
-            ));
-    }
+    @Nested
+    class AboutToSubmitCallback {
 
-    @Test
-    void shouldAddClaimIssuedDateInAboutToSubmit() {
-        CallbackParams params = callbackParamsOf(new HashMap<>(), CallbackType.ABOUT_TO_SUBMIT);
+        @Test
+        void shouldAddClaimIssuedDate_whenAboutToSubmitInvoked() {
+            CallbackParams params = callbackParamsOf(new HashMap<>(), CallbackType.ABOUT_TO_SUBMIT);
 
-        AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+            AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler
+                .handle(params);
 
-        assertThat(response.getData()).containsEntry("claimIssuedDate", LocalDate.now());
+            assertThat(response.getData()).containsEntry("claimIssuedDate", LocalDate.now());
+        }
     }
 
-    @Test
-    void shouldReturnExpectedSubmittedCallbackResponseObject() {
-        CallbackParams params = callbackParamsOf(new HashMap<>(), CallbackType.SUBMITTED);
-
-        SubmittedCallbackResponse response = (SubmittedCallbackResponse) handler.handle(params);
-
-        String documentLink = "https://www.google.com";
-        LocalDateTime serviceDeadline = LocalDate.now().plusDays(112).atTime(23, 59);
-        String formattedServiceDeadline = formatLocalDateTime(serviceDeadline, DATE_TIME_AT);
-
-        String body = format(
-            "<br />Follow these steps to serve a claim:"
-                + "\n* [Download the sealed claim form](%s) (PDF, 123KB)"
-                + "\n* Send the form, particulars of claim and "
-                + "<a href=\"%s\" target=\"_blank\">a response pack</a> (PDF, 266 KB) to the defendant by %s"
-                + "\n* Confirm service online within 21 days of sending the form, particulars and response pack, before"
-                + " 4pm if you're doing this on the due day", documentLink, responsePackLink, formattedServiceDeadline);
-
-        assertThat(response).isEqualToComparingFieldByField(
-            SubmittedCallbackResponse.builder()
-                .confirmationHeader("# Your claim has been issued\n## Claim number: TBC")
-                .confirmationBody(body)
-                .build());
+    @Nested
+    class SubmittedCallback {
+
+        @Test
+        void shouldReturnExpectedSubmittedCallbackResponse_whenSubmitted() {
```

Comment:
Maybe we should be more specific here? And not mention callback name as mentioned above, for example something like `shouldReturnConfirmationData_whenInvoked`?